### ISSUE TITLE
Partially work with native Intel HDA (ICH9)

### ISSUE THREAD

**Issue by Kitsune64** [Baron-von-Riedesel/VSBHDA#26]
Monkey Island 1: work with sound
Midier (player): work with sound
Doom: crash before booting the game. Don't know why.

I use Jemm386 and Himemx for memory management from Freedos project. Is this the cause of the issue?

**Comment by Baron-von-Riedesel** [comment]
> I use Jemm386 and Himemx for memory management from Freedos project. Is this the cause of the issue?

No. First thing to try is to use IRQ 2 - instead of 5 or 7 - for Doom. If that eliminates the crash, IRQ 5 or 7 or both are used by PCI devices.

**Comment by Kitsune64** [comment]
Hi,

I have reinstalled MS-DOS 6.22 and put this line about JEMMEX in CONFIG.SYS:
DEVICE=C:\DRIVERS\SBEMU\JEMMEX.EXE NOEMS X=TEST NOVME NOINVLPG

I have find this line in a previous install of FreeDOS where Doom can launch with sound with your emulator.

Doom work now on MS-DOS 6.22 here with VSBHDA, I have no issue about using IRQ 5 for now. But I take your advice.

  Thanks for your work. It's a well done job.
